src/System.ServiceModel.Syndication/src/System/ServiceModel/Syndication/SyndicationFeed.cs

```diff
@@ -265,6 +269,44 @@ public TextSyndicationContent Title
             set { _title = value; }
         }
 
+        internal SyndicationLink GetDocumentation()
+        {
```

Comment:
This is basically a property getter. Can you change this to a read only property with a name such as InternalDocumentation.